config/locales/en.yml

```diff
@@ -317,6 +317,8 @@ en:
         journal_status: "Journal Status"
         reference: "Reference"
         description: "Notes"
+        bulk_note_checkbox: Use Bulk Note
```

Comment:
There's a key for `facility_accounts_reconciliation:` up on line 213, that's where I'd put these